src/extension/_locales/en/messages.json

```diff
@@ -666,5 +669,14 @@
     },
     "deleting_state": {
       "message": "Deleting..."
+    },
+    "add_project": {
```

Comment:
We already have a message for `Add this project` used in the action menu. Made a new one to be inline with the designs. Same goes for `Remove project` vs `Remove this project`

Reply:
I suggest we just reuse what we have instead of adding variants with slightly different phrasing.

Reply:
Done